proxy_test.go

```diff
@@ -900,3 +901,29 @@ func TestHyperstartResponse(t *testing.T) {
 
 	rig.Stop()
 }
+
+func TestStoreRestore(t *testing.T) {
```

Comment:
Can you add some new tests where the restore fails due to an invalid state file (zero size, invalid contents (non-json, pure text (`hello world` or something ;-)))

Reply:
Sure, will do.